charts/rstudio-launcher-rbac/NEWS.md

```diff
@@ -1,3 +1,8 @@
+# 0.2.6
+
+- Update `rstudio-library` dependency. This adds `pods/exec` access to the API
+  - This is important for VSCode sessions to quit properly
```

Comment:
```suggestion
  - This is important for quitting sessions properly.
```

This involves Jupyter sessions as well so I'd be more generic about this language.